package.json

```diff
@@ -54,6 +54,7 @@
     "feathers-swagger": "^0.5.0",
     "fs-blob-store": "^5.2.1",
     "helmet": "^3.8.1",
+    "lodash": "^4.17.4",
```

Comment:
do we really need this? It looks like you're only using `_.extend`. I think we can just use es6 `Object.assign` instead. I'd rater not introduce any unnecessary dependencies.

In general, I think we should be looking at ways to remove dependencies as they are a security risk. Not so much here as in the dapp however

Reply:
No we don't I guess. I'll change it

Reply:
Fyi, lodash is already a dependency in web3. 
but I replaced it anyways